docet-core/src/main/java/docet/engine/PDFDocumentHandler.java

```diff
@@ -765,9 +765,10 @@ private void appendTOC(List<TOCNode> nodes, StringBuilder builder) {
         builder.append("<ol>");
 
         for(TOCNode node : nodes) {
+            final String levelClass = "l" + Math.min(node.document.getLevel(), 3);
```

Comment:
Shouldn't this be 'max' ?

Reply:
Nope, this is the output we want 
For level1 ->"l1"
For level 2 > "l2"
For level 3,4,5.. > "l3"

Reply:
thanks